test/RecipeBot.Domain.Test/Factories/AuthorModelFactoryTest.cs

```diff
@@ -45,9 +46,10 @@ public void Creating_model_with_author_name_with_invalid_character_length_throws
         Action call = () => factory.Create(authorData);
 
         // Assert
-        var exception = Assert.Throws<ModelCreateException>(call);
         var expectedMessage = $"AuthorName must be less or equal to {maximumAuthorNameLength} characters.";
-        Assert.Equal(expectedMessage, exception.Message);
+
+        call.Should().ThrowExactly<ModelCreateException>()
```

Comment:
Distinction between Throw and ThrowExactly -->  try to be consistent with Throw

Reply:
Use Throw when there are no derived Exception classes, ThrowExactly when there are.